rcl_lifecycle/src/com_interface.h

```diff
@@ -79,6 +79,7 @@ rcl_ret_t
 RCL_WARN_UNUSED
 rcl_lifecycle_com_interface_publish_notification(
   rcl_lifecycle_com_interface_t * com_interface,
+  const char * transition_label, uint8_t transition_id,
   const rcl_lifecycle_state_t * start, const rcl_lifecycle_state_t * goal);
```

Comment:
just a curiosity, does that make more sense if that takes `const rcl_lifecycle_transition_t * transition` which includes everything but `com_interface`?

Reply:
It would, but originally I just wanted more transparency with what I was adding, that'll be fixed.